anitya/ui.py

```diff
@@ -181,6 +185,24 @@ def projects():
     )
 
 
+def sort_projects(projects, sort):
```

Comment:
I'm thinking if it wouldn't be better to move this to `db/models.py` as it will make more sense to have this as part of [Project object](https://github.com/fedora-infra/anitya/blob/master/anitya/db/models.py#L192)

Reply:
I agree, this will make code much cleaner, I'm doing rn